stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -1,9 +1,9 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-module BLAS
-@doc """
```

Comment:
Presumably there was a reason this was how it was, so it's best to limit the changes here to just the `libblas` constant.

Reply:
The reason in the original PR that did this stated that `help>? BLAS` didn't work, so they claimed they had to do this.  With the change in this  branch to the docstring, it does indeed work,  as I have checked, and as it does in other locations that attach module docstrings like this.

Reply:
The change was made since it is consistent with how we attach docstrings in other modules, when the docstring is attached directly to the module.

Reply:
My hunch is the author of that PR forgot to import `LinearAlgebra`

Reply:
Okay, thanks for the explanation.

Reply:
NP thanks for the review